lib/cdo/shared_constants/curriculum/shared_course_constants.rb

```diff
@@ -146,6 +149,26 @@ module SharedCourseConstants
       }
     ).freeze
 
+    # Device types for course offerings
+    DEVICE_TYPES = OpenStruct.new(
+      {
+        computer: 'Computer',
+        chromebook: 'Chromebook',
+        tablet: 'Tablet',
+        mobile: 'Mobile',
+        no_device: 'No Device'
+      }
+    ).freeze
+
+    # Device compatibility levels for course offerings
+    DEVICE_COMPATIBILITY_LEVELS = OpenStruct.new(
+      {
+        ideal: 'Ideal',
+        not_recommended: 'Not recommended',
+        incompatible: 'Incompatible'
+      }
+    ).freeze
```

Comment:
similar to the previous PR, please use arrays of non-user-visible strings in the constants file, and provide a map to the user-facing, translatable values of these strings in js.